frontend/src/App.tsx

```diff
@@ -2,16 +2,19 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import { routes } from "./routes";
 import Footer from "./components/Footer/Footer";
 import Header from "./components/Header/Header";
+import WhyWorthGrid from "./components/WhyWorthGrid/WhyWorthGrid";
 import './components/Header/Header.scss';
-import "./App.css"; 
+import "./App.css";
 
 function App() {
   return (
     <Router>
-      <div className="app-container">
+      <div>
```

Comment:
Replacing ```<div className="app-container">``` with a plain `<div>` likely removes layout CSS applied to .app-container, causing layout regressions across the app.

Restore the class or move required styles accordingly. Change back to: 
```<div className="app-container">``` or update CSS to target the new structure.